### ISSUE TITLE
Mobile Safari Support

### ISSUE THREAD

**Issue by sGerli** [pebble/cloudpebble#238]
In previous versions of CloudPebble it worked in Mobile Safari, it just had an issue, no support for bluetooth keyboard arrow keys and no suport for selecting text. But now it doesn't works at all.
Issues:
- IDE Left side toolbar is missing.
- No Bluetooth keyboard arrows support.
- No text select support.
- Not able to boot the emulator, it gets stuck saying "..."

**Comment by Spacerat** [comment]
Which device and operating system version are you using?

**Comment by sGerli** [comment]
iOS 9.1 in iPad Air With Safari (for iOS).

Screenshots:

1. No toolbar sidebar
![Sidebar Bug](https://cloud.githubusercontent.com/assets/3890044/10952977/03372dfa-830b-11e5-891e-2d5f0ed0531c.PNG)

2. Emulator not booting
![Emulator Bug](https://cloud.githubusercontent.com/assets/3890044/10953134/166c4b48-830c-11e5-84d1-7f08eea768a4.PNG)

**Comment by Spacerat** [comment]
We can't seem to reproduce this! We visited CloudPebble on an actual iPad and also using the iOS simulator and didn't experience either of those problems. I'm not sure what to suggest in your case. Does the problem persist after restarting your iPad?

**Comment by yoshiboarder** [comment]
In my case, "Bluetooth keyboard arrows" doesn't work.

But other issues is working fine.

Could you please check bluetooth arrows key?

**Comment by sGerli** [comment]
Yes you are right, the other issues were caused by the adblock, but the arrow keys don't work.

**Comment by yoshiboarder** [comment]
I'm really need to this features. iPad is one of my favorite pebble app develop machine!
